tests/foreman/ui/test_organization.py

```diff
@@ -355,23 +355,120 @@ def test_negative_update(self):
 
     @run_only_on('sat')
     @tier2
-    def test_positive_remove_domain(self):
+    def test_positive_create_with_subnet(self):
+        """Create Org with subnet.
+
+        :id: 6736cd82-a2b0-4fc0-a2bc-99c9f13464d7
+
+        :assert: Org created with subnet.
+
+        :CaseLevel: Integration
+        """
+        with Session(self.browser) as session:
+            for subnet_name in generate_strings_list():
+                with self.subTest(subnet_name):
+                    # Create subnet using nailgun
+                    subnet = entities.Subnet(
+                        name=subnet_name,
+                        network=gen_ipaddr(ip3=True),
+                        mask='255.255.255.0'
+                    ).create()
+                    self.assertEqual(subnet.name, subnet_name)
+                    org_name = gen_string('alpha')
+                    make_org(session, org_name=org_name, subnets=[subnet_name])
+                    self.assertIsNotNone(self.org.search(org_name))
+                    self.org.search_and_click(org_name)
+                    session.nav.click(tab_locators['context.tab_subnets'])
```

Comment:
Suggest to use `self.org` instead `session.nav`, but both option gonna work fine